Makefile.toml

```diff
@@ -169,3 +169,17 @@ dependencies = [
 	"misc-lints-clippy-all",
 	"misc-lints-missing-docs",
 ]
+
+# Remove Cargo.lock files in cases where compilation fails
+# due to incorrect library update propogation
+[tasks.dist-clean]
+workspace = false
+script_runner = "@duckscript"
+script = [
+'''
+workspace_members = split ${CARGO_MAKE_CRATE_WORKSPACE_MEMBERS} ";"
+for workspace in ${workspace_members}
+    rm "${workspace}/Cargo.lock"
+end
+'''
+]
```

Comment:
AFAICS, running `cargo make dist-clean` won't actually (implicitly) run `cargo make clean`, which is probably the expected behavior.

Reply:
Done